cmd/zetaclientd/start.go

```diff
@@ -234,6 +229,40 @@ func start(_ *cobra.Command, _ []string) error {
 		masterLogger.Info().Msg("TSS listener received an action to shutdown zetaclientd.")
 		signalChannel <- syscall.SIGTERM
 	})
+	// debug: printout connected peers
+	go func() {
+		for {
+			time.Sleep(30 * time.Second)
```

Comment:
Maybe we could set this lower (like 1 second) to make the data more accurate for interactive debugging